ant.js

```diff
@@ -0,0 +1,161 @@
+var xoff = 0;
+var yoff = 0;
+
+function Ant(x, y, nest) {
+  this.pos = createVector(x, y);
+  this.prevPos = this.pos.copy();
+  this.vel = createVector(0, 0);
+  this.acc = createVector(0,0);
+  this.maxspeed = 1;
+  this.maxforce = 0.2;
+  this.d = 5;
+  this.inc = 0.1;
+
+  this.nestPos = createVector(nest.pos.x, nest.pos.y);
```

Comment:
Can you put this logic into the nest instead so you can pass the ant's position to the nest and the nest will figure out whether or not it is inside? (replacing the `insideNest` function here)